packages/angular-test-app/src/preview-examples/select-ng-model.ts

```diff
@@ -10,9 +10,9 @@
 import { Component } from '@angular/core';
 
 @Component({
-  selector: 'app-example',
+  selector: 'app-example-ng-model',
```

Comment:
`app-example` is used inside stackblitz as root component. Why would you change this?

Reply:
Intentionally to keep this example from being used inside StackBlitz since it is an alternative.